src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -488,9 +490,7 @@ def process_pulse_sequence(
                             )
 
                     else:  # qubit_sweeper_parameters
-                        if sweeper.qubits and sequencer.qubit in [
-                            _.name for _ in sweeper.qubits
-                        ]:
+                        if sequencer.qubit in [qubit.name for qubit in sweeper.qubits]:
```

Comment:
Ok, the usage of `_` as a variable was terrible (conventionally is a black hole, though it is a valid variable name), and I expect `sweeper.qubits` to be just a list, so this is definitely more minimal than the one before (if is an empty list, even the comprehension will result in an empty one, and thus `in` will return `False` in any case).

But it should be a nice refactoring, without any consequence on execution, isn't it?

Reply:
> But it should be a nice refactoring, without any consequence on execution, isn't it?

Actually the previous condition was so hard to read that I thought it was wrong, but now that I look at it again indeed what I am doing should be equivalent (but hopefully more readable).

Reply:
Ok, let's assume this is the case, and keep the other.